src/wallet/wallet.cpp

```diff
@@ -1840,7 +1860,7 @@ bool CWalletTx::IsTrusted(interfaces::Chain::Lock& locked_chain) const
     if (!locked_chain.checkFinalTx(*tx)) {
```

Comment:
Note for future followup: I believe we could call [`IsFinalTx`](https://github.com/bitcoin/bitcoin/blob/50591f6ec61b802cf4193cdbefcc85ad75716e8d/src/consensus/tx_verify.cpp#L17-L28) instead of [`checkFinalTx`](https://github.com/bitcoin/bitcoin/blob/50591f6ec61b802cf4193cdbefcc85ad75716e8d/src/validation.cpp#L189-L219) so `locked_chain` argument no longer needs to be passed to `CWalletTx::IsTrusted`